pkg/virt-handler/virtwrap/cli/libvirt.go

```diff
@@ -318,29 +317,28 @@ type VirDomain interface {
 	Free() error
 }
 
-func waitForLibvirt(uri string, user string, pass string, timeout time.Duration) error {
-	interval := 10 * time.Second
-	return utilwait.PollImmediate(interval, timeout, func() (done bool, err error) {
-		if virConn, err := newConnection(uri, user, pass); err == nil {
-			defer virConn.Close()
-			return true, nil
-		}
-		return false, nil
-	})
-}
-
 func NewConnection(uri string, user string, pass string, checkInterval time.Duration) (Connection, error) {
 	timeout := 15 * time.Second
+	interval := 10 * time.Second
```

Comment:
Since you're in here fixing things, would you mind making `timeout` and `interval` be module level constants (possibly with a better name). It's just good form to keep arbitrary numbers highly visible.

Reply:
Already on it.